raiden/api/python.py

```diff
@@ -645,6 +652,26 @@ def transfer_async(
         if not is_binary_address(target):
             raise InvalidAddress('target address is not valid.')
 
+        if secret is not None:
+            if len(secret) != 66:
```

Comment:
Please put this along with your description of how this was derived in https://github.com/raiden-network/raiden/blob/master/raiden/constants.py

Reply:
Done